package.json

```diff
@@ -41,38 +41,37 @@
     "primeng": "^10.0.0",
```

Comment:
PrimeNG needs to be updated in the same way as Angular to v12

Reply:
The required dependencies are   "primeicons": "^4.1.0", and "primeng": "^12.0.1",